docs/pages/server-access/guides/openssh.mdx

```diff
@@ -23,12 +28,45 @@ We've outlined these reasons in [OpenSSH vs Teleport SSH for Servers?](https://g
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-- A Linux host with the OpenSSH server `sshd` installed, but not Teleport. The
-  SSH port on this host must be open to traffic from the Teleport Proxy Service
-  host.
+- A Linux host with the OpenSSH server `sshd` version 7.4 or above installed,
+  but not Teleport. The SSH port on this host must be open to traffic from the
+  Teleport Proxy Service host.
 - (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/4. Configure `sshd` to trust the Teleport CA
+## Step 1/5. Add a node resource to your Teleport cluster
+
+When you request an SSH connection to a OpenSSH node, Teleport needs to be able
+to find the node's IP address so it can establish a connection to it.
+
+Declare a `node` resource so Teleport knows how to reach your OpenSSH server.
+On your local machine, create a file called `openssh-node-resource.yaml` with the following content:
+
+```yaml
+kind: node
+version: v2
+sub_kind: openssh
+metadata:
+  name: a100fdd0-52db-4eca-a7ab-c3afa7a1564a
+  labels:
+    env: prod
+spec:
+  addr: 1.2.3.4:22
+  hostname: openssh-node
+```
+
+The fields `spec.addr` and `spec.hostname` are mandatory, but `metadata.labels` is not.
+The `metadata.name` field isn't mandatory, but setting it here will save you some work later.
```

Comment:
```suggestion
`spec.addr` and `spec.hostname` are mandatory. Assign `spec.addr` to the address and port of your node and `spec.hostname` to the name of the node as you would like users to see it in Teleport.

 The `metadata.labels` field labels the SSH Service instance so you can apply RBAC rules to it. 

The `metadata.name` field isn't mandatory, but setting it here will save you some work later.
```

I'm not sure we should assume that a reader will correctly guess the meanings of configuration fields.

Reply:
Yeah that's much better, done